pkgs/sdk/server/src/Integrations/DataSystemComponents.cs

```diff
@@ -0,0 +1,47 @@
+using System;
+using LaunchDarkly.Sdk.Server.Subsystems;
+
+namespace LaunchDarkly.Sdk.Server.Integrations
+{
+    /// <summary>
+    /// Components for use with the data system.
+    /// <para>
+    /// This class is not stable, and not subject to any backwards compatibility guarantees or semantic versioning.
+    /// It is not suitable for production usage. Do not use it. You have been warned.
+    /// </para>
+    /// </summary>
+    internal static class DataSystemComponents
```

Comment:
It did not feel reasonable to add this to the ever growing list of "Components".